QModPluginEmulator/QModPluginGenerator.cs

```diff
@@ -70,6 +82,21 @@ public static void Finish()
             }
         }
 
+        public static List<string> DirtyLogStrings = new List<string>()
+        {
+            "(Filename", "Resetting cell with", "Replacing cell",
+            "PerformGarbage", "Fallback handler could not load"
+        };
+
+        private static void LibcHelper_Format_Postfix(ref string __result)
+        {
+            foreach(string dirtyString in DirtyLogStrings)
+            {
+                if(__result.Contains(dirtyString))
```

Comment:
Use [string.StartsWith](https://docs.microsoft.com/en-us/dotnet/api/system.string.startswith?view=net-5.0) instead of `string.Contains` to keep this from getting slower than it has to be.

Reply:
Or `IndexOf` can be used from the start

Reply:
problem is not all of these are at the start. that stupid filename thing is actually stuck to the end of a prev line...

Reply:
I mean, you're using 'IndexOf' in the next line anyway, it's a double work.

Reply:
how about this 
83ae57c

Reply:
Do you know the first rule of optimization? In this case, any string comparisons in memory are nothing compared to writing log message to disk.

Reply:
That's a fair assessment given the ratio between "number of log lines" vs "number of frames drawn".

We can call it good here and expand as needed later.